features/minion/core/impl/blueprint-heartbeat.xml

```diff
@@ -0,0 +1,53 @@
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
+	xmlns:ext="http://aries.apache.org/blueprint/xmlns/blueprint-ext/v1.1.0"
+	xsi:schemaLocation="
+		http://www.osgi.org/xmlns/blueprint/v1.0.0 
+		http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd
+		http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0
+		http://aries.apache.org/schemas/blueprint-cm/blueprint-cm-1.1.0.xsd
+		http://aries.apache.org/blueprint/xmlns/blueprint-ext/v1.1.0
+		http://aries.apache.org/schemas/blueprint-ext/blueprint-ext-1.1.xsd
+">
+
+	<cm:property-placeholder id="minionHeartBeatProperties"
+		persistent-id="org.opennms.minion.controller" placeholder-prefix="[["
+		placeholder-suffix="]]" update-strategy="none">
+		<cm:default-properties>
+			<cm:property name="location" value="MINION" />
+			<cm:property name="id" value="00000000-0000-0000-0000-000000ddba11" />
+		</cm:default-properties>
+	</cm:property-placeholder>
+
+
+	<!-- ActiveMQ component provided by minion-core -->
+	<reference id="queuingservice" interface="org.apache.camel.Component"
+		filter="(alias=opennms.broker)" />
+
+	<bean id="marshaller" class="org.opennms.core.camel.JaxbUtilsMarshalProcessor">
+		<argument value="org.opennms.minion.core.api.MinionIdentityDTO" />
+	</bean>
+
+	<!-- A canonical source for this Minion's identity -->
+	<bean id="minionIdentity" class="org.opennms.minion.core.api.MinionIdentityDTO">
```

Comment:
We should build the DTO using the MinionIdentity class pulled from the service registry instead of referencing the properties again.